.markdownlint-cli2.jsonc

```
@@ -87,6 +86,7 @@
     "./markdownlint/TOP006_fullFencedCodeLanguage/TOP006_fullFencedCodeLanguage.js",
     "./markdownlint/TOP007_useMarkdownLinks/TOP007_useMarkdownLinks.js",
     "./markdownlint/TOP008_useBackticksForFencedCodeBlocks/TOP008_useBackticksForFencedCodeBlocks.js",
-    "./markdownlint/TOP010_useLazyNumbering/TOP010_useLazyNumbering.js"
+    "./markdownlint/TOP009_lessonOverviewItemsSentenceStructure/TOP009_lessonOverviewItemsSentenceStructure.js",
+    "./markdownlint/TOP010_useLazyNumbering/TOP010_useLazyNumbering.js",
```

JSON doesn't support trailing commas, hence the syntax error being raised
```suggestion
    "./markdownlint/TOP010_useLazyNumbering/TOP010_useLazyNumbering.js"
```